### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (12 - Captions)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#491]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

7.3 captions
 
The first line is again labeled as a guideline that reads "Provide captions and associated metadata for audio content".   
•	Because is non-normative, it does not need to be testable.  But again I would suggest thinking about making these Out-come oriented – or dropping them and going straight to the Outcome.  The imperative is confusing if it is not normative. 
•	The phrase “captions and associated metadata" should be a link so that someone can go find out what that is.  Metadata is very clearly not a common word and very few people actually know what it means.   (I am actually looking forward to a plain language explanation of metadata that is clear and easy to understand. I remember when I first encountered the word and tried to figure out what it meant many decades ago…) 
The outcome then is "translate speech and nonspeech audio”
•	First of all, this is not outcome-oriented unless the intent is that the page literally is a page that translates speech and nonspeech audio.  That is, it literally is a page you talk to and it will translate speech, and also listen to all the audio in the background, and translate that as well.
•	Since that is not the intent, I presume this means something like “provides a translation of speech and non-speech audio”.  But that would be imperative and not outcome-oriented.
•	Perhaps it meant to say, “A translation of speech and nonspeech audio is available". 
o	This would be an outcome and would be testable if they defined what translate means.  To me translation usually refers to changing something into another language.  The description below sort of explains what it means but still uses the word “translates”.  
o	If the text below is a rationale, it's not written as a rationale but rather it's written in a longer form of the outcome.  These two should not be confused or conflated.  Only the phrase above is marked-up as being the outcome.  The phrase below is… I'm not sure what.  
o	Also – I note that the last text goes on to say "user agents in APIs support the display and control of captions".  
	It's not clear if that is a different requirement? 
	Is that a different or additional outcome that is expected - than what is specified in outcome? 
	I would note that these is not things that are under the control of the author (user agents and APIs).  So, it can't be an outcome that the author is responsible for.  
	Is this simply a  "and by the way"  comment ?   Or is it meant to be a separate outcome that should be listed as a separate outcome for a different audience than the author?  If so, should it be a separate provision and tagged as applying only to user agent and API generators?

I would suggest something like the following
1.	Consider dropping the guideline imperative
2.	Change the provision to be outcome-oriented,  something like 
a.	“Speech and non-speech audio is available in a visual form, and in a form that is readable by assistive technologies. 
3.	Change the rationale to something like: 
a.	Making the audio available in visual form makes it available to people who cannot hear or understand the audio for some reason.  Making it available in a form that is readable by assistive technologies, allows others who cannot see or hear, or who cannot hear well enough or read well enough to have the content transformed into a form they can perceive and understand – including translating it into other languages or, in the future, into simpler language. 
4.	Move any requirement for user agents and API creators to a separate provision – that is marked as applying only to them.
I also note that the last paragraph says e.g. captions.  This is fine if this is a rationale.  If this was meant to be normative, then examples are not allowed.  The statement needs to say exactly what is required.  If an example is provided in the normative statement, then it is not clear if that is sufficient (e.g. nothing else needs to be done) or it is just one of a longer (unspecified therefore unknown to the author) list of things that need to be done. Do I also need to translate it into braille and symbols and other languages? 
The outcome or objective part of this would similarly need to tell people exactly what formats are required or not required.  Otherwise, it's untestable. The author is unable to determine if they pass or fail.  And it would be impossible for others to judge reliably and consistently whether it passes or fails.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
